tools/php_api_ref/phpdoc.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+AUTH_JSON=$1;
+OUTPUT_DIR=$2;
+
+FLAVOR='commerce';
+VERSION='4.4.*';
```

Comment:
```suggestion
VERSION='4.5.*';
```

I locally use `VERSION='4.5.*@dev';` to get the beta versions until first release.